.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,22 @@
+# https://pre-commit.com
+# This GitHub Action assumes that the repo contains a valid .pre-commit-config.yaml file.
+# Using pre-commit.ci is even better that using GitHub Actions for pre-commit.
```

Comment:
Minor typo: `than using` instead of `better that using`
Not sure this line makes sense in the code though. It would be rather a node for me inside the PR's description.

Reply:
If a repo administrator adds this repo to https://pre-commit.ci then we can remove this file and fixes done by pre-commit can be automatically added to pull requests.

Reply:
Ah ok that makes sense. Did understand the sentence differently.